python/fusion_engine_client/analysis/data_loader.py

```diff
@@ -649,14 +649,15 @@ def get_log_reader(self) -> MixedLogReader:
     def get_input_path(self):
         return self.reader.input_file.name
 
-    def convert_to_p1_time(self,
-                           times: Union[Iterable[Union[datetime, gpstime, Timestamp, float]],
-                                        Union[datetime, gpstime, Timestamp, float]],
-                           assume_utc: bool = False) ->\
+    def _convert_time(self, gps_to_p1,
```

Comment:
I'd suggest we make this a pair of enum or a string args `source_type` and `target_type` instead: `source_type='p1', target_type='gps'`, etc. Having `gps_to_p1=False` mean that it will convert p1->gps instead is a little unintuitive, and in the future we plan to also add conversion between system time and P1/GPS time.